share/sv.po

```diff
@@ -0,0 +1,76 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: 1.0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-11-16 17:49+0100\n"
+"PO-Revision-Date: 2021-11-16 17:49+0100\n"
+"Last-Translator: mats.dufberg@iis.se\n"
+"Language-Team: Zonemaster project\n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "Domain name required"
+msgstr "Domännamn är obligatoriskt"
+
+msgid ""
+"The domain name is not a valid IDNA string and cannot be converted to an A-"
+"label"
+msgstr ""
+"Domännamnet är inte ett giltigt IDN-namn och kan inte konverteras till A-"
+"labelformat"
```

Comment:
Is it okay to have a compound word that switches languages half way through? Wouldn't it be better with "A-label-format"?

Reply:
Yes, but it depends on how well-known the expression is. In this case an extra hyphen can be motivated. I'll update.